automation/test.sh

```diff
@@ -428,6 +431,8 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} && -z ${label_filter}
     label_filter='(sig-storage)'
   elif [[ $TARGET =~ wg-s390x ]]; then
     label_filter='(wg-s390x)'
+  elif [[ $TARGET =~ wg-arm64 ]]; then
+    label_filter='(wg-arm64 && !(ACPI,software-emulation,requires-two-schedulable-nodes,cpumodel))'
```

Comment:
Are we sure we want to skip these labels?
Is it possible to instead have the `wg-arm64` label only whenever necessary?

Reply:
Hi, @iholder101, great suggestion! I believe this is definitely worth discussing.

Due to hardware(2 Arm64 servers in CI/CD pipline) limitations, we are only able to run core E2E tests on the Arm64 platform. After some evaluation, I consider vmi_lifecycle_test.go particularly important, as it tests whether KubeVirt can effectively manage VMIs. Most tests in this file should be CPU architecture-independent. My goal is to ensure that the majority of these tests pass on the Arm64 platform, and I’ve also selected a few other critical tests from other files.

However, within vmi_lifecycle_test.go, there are certain tests that cannot be executed on Arm64 due to hardware or test environment limitations. We previously discussed how to select tests for Arm64 with @rmohr, and considered the following options:

1. Tag each individual test precisely with the [arm64] label (similar to your suggestion).
2. Use a skip filter to exclude specific tests (deprecated).
3. Use a separate tag to label groups of tests by feature, and then skip them in one place (the current implementation).
Each approach has its pros and cons.

From my perspective, the third method has the following points: 
**Cons:**
- It may lead to some fragmentation of labels, but since we only apply it to vmi_lifecycle_test.go, it remains manageable.

**Pros:**
- It’s clear why certain tests are skipped for the Arm64 platform in vmi_lifecycle_test.go. This transparency helps anyone contributing to the Arm64 platform understand the reasoning.
- If new issues arise in vmi_lifecycle_test.go, we’ll know whether they can pass on the Arm64 platform, making future troubleshooting easier.